.pre-commit-config.yaml

```
@@ -0,0 +1,7 @@
+# .pre-commit-config.yaml
+repos:
+  - repo: https://github.com/c4urself/bump2version
+    rev: v1.0.1
+    hooks:
+      - id: bump2version
+        args: [patch]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)